README.md

```diff
@@ -22,18 +22,20 @@ Installation
 
 2. Install Azure Python SDKs.
 
-    Several reasons for installing Python SKDs here.
+    Several reasons for installing Python SDKs are listed here.
 
     - New module is added to the role and this module is for one new Azure service, which is not part of existing Ansible release yet. Corresponding SDK of this new service needs to be installed.
 
-    - Newer version of SDK introduces breaking API change. One specific working version should be installed here.
+    - Newer version of SDKs introduces breaking API change. One specific working version should be installed here.
```

Comment:
introduce

Reply:
i would change to "Newer version of SDK introduces"